products/jira.md

```diff
@@ -170,6 +314,405 @@ releases:
+
+-   releaseCycle: "7.0"
+    releaseDate: 2015-10-01
+    eol: 2017-10-06
+    latest: "7.0.11"
+    latestReleaseDate: 2016-09-13
+    link:
+      https://confluence.atlassian.com/display/JIRASOFTWARE/JIRA+Software+7.0.x+release+notes#JIRASoftware7.0.xreleasenotes-7-0-11
+
+-   releaseCycle: "6.4"
+    releaseDate: 2015-04-07
+    eol: 2017-03-17
+    latest: "6.4.14"
+    latestReleaseDate: 2016-07-28
+    link:
+      https://confluence.atlassian.com/jira064/jira-6-4-14-release-notes-834232021.html
+
+-   releaseCycle: "6.3"
+    releaseDate: 2014-07-09
+    eol: 2016-07-08
+    latest: "6.3.15"
+    latestReleaseDate: 2014-08-11
+    link:
+      https://confluence.atlassian.com/jira063/jira-6-3-15-release-notes-707625090.html
+
+-   releaseCycle: "6.2"
+    releaseDate: 2014-02-25
+    eol: 2016-06-11
+    latest: "6.2.7"
+    latestReleaseDate: 2014-06-11
+    link:
+      https://confluence.atlassian.com/jira063/jira-6-2-7-release-notes-683542003.html
+
+-   releaseCycle: "6.1"
+    releaseDate: 2013-09-24
+    eol: 2016-05-22
+    latest: "6.1.9"
+    latestReleaseDate: 2014-10-24
+    link:
+      https://confluence.atlassian.com/jira063/jira-6-1-9-release-notes-683542023.html
+
+-   releaseCycle: "6.0"
+    releaseDate: 2013-05-21
+    eol: 2015-09-03
+    latest: "6.0.8"
+    latestReleaseDate: 2013-09-03
+    link:
+      https://confluence.atlassian.com/jira064/jira-6-0-8-release-notes-720413919.html
+
+-   releaseCycle: "5.2"
+    releaseDate: 2012-11-12
+    eol: true
+    latest: "5.2.11"
+    latestReleaseDate: 2013-04-19
+    link:
+      https://confluence.atlassian.com/jira063/jira-5-2-11-release-notes-683542112.html
+
+-   releaseCycle: "5.1"
+    releaseDate: 2012-07-09
+    eol: true
+    latest: "5.1.8"
+    latestReleaseDate: 2012-10-31
+    link:
+      https://confluence.atlassian.com/jira063/jira-5-1-8-release-notes-683542060.html
+
+-   releaseCycle: "5.0"
+    releaseDate: 2012-02-22
+    eol: 2012-07-01
+    latest: "5.0.7"
+    latestReleaseDate: 2012-07-01
+    link:
+      https://confluence.atlassian.com/jira063/jira-5-0-7-release-notes-683542076.html
+
+-   releaseCycle: "4.4"
+    releaseDate: 2011-08-02
+    eol: 2014-02-22
+    latest: "4.4.5"
+    latestReleaseDate: 2012-02-22
+    link:
+      https://confluence.atlassian.com/jira063/jira-4-4-5-release-notes-683541908.html
+
+-   releaseCycle: "4.3"
+    releaseDate: 2011-03-16
+    eol: 2013-05-27
+    latest: "4.3.4"
+    latestReleaseDate: 2011-05-27
+    link:
+      https://confluence.atlassian.com/jira063/jira-4-3-4-release-notes-683541993.html
+
+-   releaseCycle: "4.2"
+    releaseDate: 2010-10-21
+    eol: 2013-02-08
+    latest: "4.2.4"
+    latestReleaseDate: 2011-02-08
+    link:
+      https://confluence.atlassian.com/jira063/jira-4-2-4-release-notes-683541978.html
+
+-   releaseCycle: "4.1"
+    releaseDate: 2010-04-08
```

Comment:
```suggestion
    releaseDate: 2010-04-07
```

According to https://confluence.atlassian.com/jira063/jira-4-1-release-notes-683541944.html.